gradle-plugin/README.md

```diff
@@ -464,6 +464,43 @@ Additionally, `ANDROID_SDK_ROOT` must be set and point to the Android SDK locati
 1. Tag version `gradle-plugin-vX.Y.Z`
 1. Release title `Gradle Plugin vX.Y.Z`
 1. Paste the content from `/gradle-plugin/CHANGELOG.md` as the description
+1. Bump version in `/gradle/libs.versions.toml`, add `-SNAPSHOT`, commit the changes with the message "Prepare for next development iteration" and push
 
 The `gradle-plugin-release` workflow will automatically publish the new version to the Gradle Plugin
 portal upon new release publish.
+
+### Publishing snapshots
+
+Snapshots are automatically published from `main` whenever the version in
+`/gradle/libs.versions.toml` contains `-SNAPSHOT`.
```

Comment:
I saw the URL switch in the gradle file but I couldn't see anything that stops the github action from running. Is just that one enough to stop this?

Reply:
Good point. We discussed this in slack. The snapshot portal will not allow publication of non-snapshot versions so it will fail.